### PR TITLE
Removed deprecated pulpcore.plugin.models.UnsupportedDigestValidation…

### PR DESCRIPTION
**Pull request by ggainey**
…Error.

closes #8169

### CODE REVIEW

File: CHANGES/plugin_api/8169.removal [hunk 1]
```diff
@@ -0,0 +1,2 @@
+pulpcore.plugin.models.UnsupportedDigestValidationError has been removed.
+Plugins should look for this at pulpcore.plugins.exceptions.UnsupportedDigestValidationError instead.
```
Comment (daviddavis) [inline CHANGES/plugin_api/8169.removal:2]:
```suggestion
``pulpcore.plugin.models.UnsupportedDigestValidationError`` has been removed. Plugins should look
for this at ``pulpcore.plugins.exceptions.UnsupportedDigestValidationError`` instead.
```

**Review by daviddavis — APPROVED** [review]
One suggestion on the changelog. Other than that, LGTM.

File: CHANGES/plugin_api/8169.removal (hunk 1 above)
Comment (ggainey) [reply CHANGES/plugin_api/8169.removal]: Great catch, thanks

File: pulpcore/plugin/models/__init__.py [hunk 2]
```diff
@@ -1,9 +1,6 @@
 # Models are exposed selectively in the versioned plugin API.
 # Any models defined in the pulpcore.plugin namespace should probably be proxy models.
 
-# THIS WILL BE DEPRECATED In 3.10 - LOOK TO .plugins.exceptions INSTEAD
```
Comment (mdellweg) [inline pulpcore/plugin/models/__init__.py:4]: I guess this string was wrong, as it _is_ deprecated and will be _removed_ in 3.10. :)

**Review by mdellweg — APPROVED** [review]

**Review by goosemania — APPROVED** [review]

**Comment by goosemania** [comment]
@ggainey , feel free to rebase, should pass then

**Pull request merged**